crates/isograph_lang_parser/src/parse_iso_literal.rs

```diff
@@ -563,6 +563,27 @@ fn parse_non_constant_value(
             }))
         })?;
 
+        to_control_flow::<_, WithSpan<IsographLiteralParseError>>(|| {
+            let open = tokens
+                .parse_token_of_kind(IsographLangTokenKind::OpenBrace)
+                .map_err(|with_span| with_span.map(IsographLiteralParseError::from))?;
+
+            let entries = parse_delimited_list(
```

Comment:
it's a bit weird that `parse_delimited_list` doesn't also parse the opening brace /shrug